src/js/control-bar/progress-control/seek-bar.js

```diff
@@ -38,7 +38,21 @@ class SeekBar extends Slider {
    */
   constructor(player, options) {
     super(player, options);
+    this.setEventHandlers_(player, options);
+  }
 
+  /**
+   * Sets the event handlers
+   *
+   * @param {Player} player
+   *        The `Player` that this class should be attached to.
+   *
+   * @param {Object} [options]
+   *        The key/value store of player options.
+   *
+   * @private
+   */
+  setEventHandlers_(player, options) {
```

Comment:
I would consider removing passing in the player and options and instead update the code to use `this.player_` and `this.options_`

Reply:
sure thing.